### PR TITLE
fix: assign role to user issue

### PR DESCRIPTION
**Pull request by vivekp8000**
## Change Description
  fix: assign role to user issue
Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist: (Check only applicable)

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/app/pages/manage/users/users-list/users-list.component.ts [hunk 1]
```diff
@@ -201,18 +222,17 @@ export class UsersListComponent implements OnInit {
     }
   }
 
-  async deleteUser(contactID) {
+  deleteUser(contactID, firstName: string, lastName: string) {
```
Comment (Daljit-fleetHawks) [inline src/app/pages/manage/users/users-list/users-list.component.ts]: why did you change delete user function? @vivekp8000

**Review by kunalfleethawks — APPROVED** [review]

**Review by Daljit-fleetHawks — APPROVED** [review]
